Common/GPU/Vulkan/VulkanContext.cpp

```diff
@@ -132,7 +133,7 @@ VkResult VulkanContext::CreateInstance(const CreateInfo &info) {
 #endif
 #endif
 
-	if (flags_ & VULKAN_FLAG_VALIDATE) {
+	if (flags_ & VULKAN_FLAG_VALIDATE && g_Config.customDriver.empty()) {
```

Comment:
This expression is better written with parenthesis around the & operator subexpression. I'm always suspicious of operator precedence around binary operators vs logical ones..